config/locales/en.yml

```diff
@@ -45,3 +45,8 @@ en:
   administrate:
     actions:
       new: 'New'
+  podcasts:
+    statuses:
+      unreachable: Podcast's feed_url is not reachable
```

Comment:
Used locales because it seems like a natural way to keep such data, though locales are not widely used in the app.

Reply:
Nice